packages/panzer/disc-fe/src/evaluators/Panzer_GatherSolution_Epetra_decl.hpp

```diff
@@ -126,7 +126,9 @@ class GatherSolution_Epetra<panzer::Traits::Residual,TRAITS,LO,GO>
   bool useTimeDerivativeSolutionVector_;
   std::string globalDataKey_; // what global data does this fill?
 
-  Teuchos::RCP<Epetra_Vector> x_;
+  Teuchos::RCP<Epetra_Vector>                    x_;
+  Teuchos::RCP<const Thyra::VectorBase<double> > xOwned_;
+  Teuchos::RCP<Epetra_Vector>                    xGhosted_;
```

Comment:
The goal is to eventually just have `xOwned_` and `xGhosted_`, with the latter containing no duplicate information from the former.  For the time being, `x_` is retained so that we can develop this alternate way of doing things without breaking what's currently working.